Corona-Warn-App/src/main/res/values/strings.xml

```diff
@@ -614,7 +615,7 @@
     <!-- XACT: settings(tracing) - describes illustration -->
     <string name="settings_tracing_bluetooth_illustration_description_inactive">"A person has turned off Bluetooth on their smartphone, so an encounter with two other persons is not logged."</string>
     <!-- XACT: settings(tracing) - describes illustration -->
-    <string name="settings_tracing_location_illustration_description_inactive"/>
+    <string name="settings_tracing_location_illustration_description_inactive" />
```

Comment:
Are the additional space intended?

Reply:
No, it's probably the result of auto formatting. Looks like it wants to do this only to strings that are closed by `/>` but not ones that are closed by `</string>` 😄

Reply:
ah okay, thanks! Then we should leave it as it is :D